### PR TITLE
Documentation for setting up elk and utilising it with filebeat to ca…

### PR DESCRIPTION
**Pull request by sarvaniy**
…pture the log files metrics

Signed-off-by: sarvaniy <sarvanimounika@gmail.com>

### CODE REVIEW

**Review by ChaitanyaKaranam — COMMENTED** [review]
Looks good for me.

File: projects/elk/filebeat/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    && cp filebeat /bin \
+    && mkdir -p /etc/filebeat \
+    && cp filebeat.yml /etc/filebeat/filebeat.yml \
+    && rm -rf /tmp/filebeat*
+```
+### configure filebeat
+
+configure the set of required log files to capture in elasticsearch index
+
+Contact logstash for parsing and storing the logs in elasticsearch index, so mention logstash section in filebeat.yml
```
Comment (tandonri) [inline projects/elk/filebeat/README.md]: Please have a numbered list for ordered steps.

**Review by tandonri — COMMENTED** [review]
Please have a numbered list for ordered steps in ``elk/README.md``

File: projects/elk/filebeat/README.md (hunk 1 above)
Comment (sarvaniy) [reply projects/elk/filebeat/README.md]: I made the changes and committed as well.

**Review by ChaitanyaKaranam — APPROVED** [review]

**Pull request merged**